src/main/rx/sumd.c

```diff
@@ -139,10 +137,9 @@ static uint8_t sumdFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
             return frameStatus;
     }
 
-    if (sumdChannelCount > MAX_SUPPORTED_RC_CHANNEL_COUNT)
-        sumdChannelCount = MAX_SUPPORTED_RC_CHANNEL_COUNT;
+    unsigned channelsToProcess = (sumdChannelCount > MAX_SUPPORTED_RC_CHANNEL_COUNT) ? MAX_SUPPORTED_RC_CHANNEL_COUNT : sumdChannelCount;
```

Comment:
Why not:
```
    unsigned channelsToProcess = MIN(sumdChannelCount, MAX_SUPPORTED_RC_CHANNEL_COUNT);
```

Reply:
Answer: Insufficient level of caffeine in bloodstream when writing this. Fixed.